app/src/main/res/layout-land/profile_progress_header.xml

```diff
@@ -51,6 +51,7 @@
       android:text="@{viewModel.profile.name}"
       android:textColor="@color/oppiaPrimaryText"
       android:textSize="28sp"
+      android:textAlignment="viewStart"
```

Comment:
I don't think this is needed, try to check app without using this parameter.

Reply:
ok

Reply:
@lakshaygarg128 You have replied done, but its not reverted?

Reply:
@rt4914  Sorry Sir , I have a doubt while  testing all the data saved as String is working properly in rtl support but the username that is from backend is not working properly in rtl support . Although the alignment in landscape is done .
Here the Admin is not same like profile .


<img width="427" alt="test" src="https://user-images.githubusercontent.com/55504132/103613372-79880c00-4f4c-11eb-8a49-08db1432794e.png">